strings/aho_corasick.py

```diff
@@ -61,7 +62,7 @@ def set_fail_transitions(self):
                     + self.adlist[self.adlist[child]["fail_state"]]["output"]
                 )
 
-    def search_in(self, string):
+    def search_in(self, string: str) -> Dict:
```

Comment:
```suggestion
    def search_in(self, string: str) -> Dict[str, List[int]]:
```
Please be more specific